packages/styles/src/atoms/colours.ts

```diff
@@ -70,6 +70,8 @@ export const HIGHLIGHT_GREEN = colours.NHS_HIGHLIGHT_GREEN
 export const TRANSLUCENT_BRIGHT_BLUE_TABLE = colours.NHS_TRANSLUCENT_BRIGHT_BLUE_TABLE
 export const TRANSLUCENT_GREY_TABLE = colours.NHS_TRANSLUCENT_MID_GREY_TABLE
 
+export const GREY = 'rgb(190, 190, 190)'
```

Comment:
I'm thinking out loud here but is it worth using one of these existing greys from `nhs-colours.ts`?
![image](https://user-images.githubusercontent.com/63285990/199478575-b8475a91-34f0-4df9-951b-a04184e5e9d0.png)
I don't know if they come from any official style guide - possibly https://service-manual.nhs.uk/design-system/styles/colour

Reply:
how about using this from the nhs palette 
![image](https://user-images.githubusercontent.com/115567223/199479185-4eb90c2c-7287-44d4-a8c7-ec044351fb72.png)

Reply:
Yep looks good to me :)